packages/vite-plugin/src/main.ts

```diff
@@ -384,7 +384,7 @@ export default ({
           },
         },
         server: {
-          port: 9000,
+          port: config?.server?.port || 9000,
```

Comment:
We should use nullish coalescing whenever applicable:
```suggestion
          port: config?.server?.port ?? 9000,
```
I know the port can't be `0` in this specific case but `??` better conveys the meaning of the code (if `port` is not defined, use `9000`)